src/DwitTech.AccountService.Core/Templates/WelcomeEmail.html

```diff
@@ -0,0 +1,28 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <title>Welcome Email</title>
+</head>
+<body>
+    <h2>Dear {{Firstname}} {{Lastname}},</h2>
+
+    <p>
+        Welcome on board. Your account has been successfully set up and you can begin using it.
+
+        You will need the login details below to manage your dashboard
+        <br />
+        Email: {{Email}}
+        Password: {{Password}}
```

Comment:
Will we have the password?
If the password will be hashed before storage, then I don't think this line is necessary because we can't send the user his hashed password, and even if we did had the original password, it would probably be a huge security risk sending it to the user via mail.

Reply:
> Will we have the password? If the password will be hashed before storage, then I don't think this line is necessary because we can't send the user his hashed password, and even if we did have the original password, it would probably be a huge security risk sending it to the user via mail.

Resolved